module1/homework/removeVowels/vowel.cpp

```diff
@@ -0,0 +1,15 @@
+#include "vowel.hpp"
+
+#include <algorithm>
+
+std::vector<std::string> removeVowels(std::vector<std::string>& vec) {
+    std::vector<char> vowelVec{'a', 'e', 'i', 'o', 'u', 'y'};
+
+    for (std::string& str : vec) {
+        for (const char& vowel : vowelVec) {
+            str.erase(std::remove(str.begin(), str.end(), vowel), str.end());
+            str.erase(std::remove(str.begin(), str.end(), vowel - 32), str.end());
```

Comment:
redundant line and you use magic number

Reply:
It isn't a redundant line. This magic number 32 is a difference between lowcases and uppercases in ASCII table - so here we have removing of uppercases. :D Anyway it's harder to understand, and for me it's better type uppercases in containers with vowels.

Reply:
I know but if you using my suggestion with array of vowels you don't need to use erase function twice :)

Reply:
Generally we are trying to avoid using raw numbers in the code(a.k.a magic numbers). We are using variables for it( like constexpr int asciiDiff = 32 ) . That makes code more readable and makes change of this number easier.